robot/rospackages/src/ping_acknowledgment/scripts/ping_response_client.py

```diff
@@ -3,54 +3,59 @@
 import sys
 import datetime
 import rospy
-from std_msgs.msg import Float32
 from ping_acknowledgment.srv import *
 
+#
+# ping_response_client
+# 
+# Use without arguments to obtain detailed timestamps and latency data
+# Use option "api" to just receive the time it took in millisecond for the ping.
+#
 
-def ping_response_client(msg):
+
+def ping_server():
```

Comment:
Why is this renamed to `ping_server`, isn't this the client script?

Reply:
to send a ping to the server